### PR TITLE
Moves datastore intializations into setUp function in testing file and increases testing file's readability.

### PR DESCRIPTION
**Pull request by meagancheese**

### CODE REVIEW

**Review by apluscs — APPROVED** [review]
LGTM. Just curious why was this done? Did the other way cause bugs?

File: src/test/java/com/google/sps/data/DatastoreAvailabilityDaoTest.java [hunk 1]
```diff
@@ -43,8 +43,8 @@
   private final LocalServiceTestHelper helper =
       new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
 
-  private final DatastoreAvailabilityDao tester = new DatastoreAvailabilityDao();
-  private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+  private DatastoreAvailabilityDao tester;
```
Comment (GrantComm) [inline src/test/java/com/google/sps/data/DatastoreAvailabilityDaoTest.java]: Sorry I missed this before but as previously requested by Jack, change name from "tester" to something more descriptive such as "dao"

**Review by GrantComm — CHANGES_REQUESTED** [review]

**Comment by meagancheese** [comment]
@apluscs , no bugs, but Jack was very surprised that there weren't and so asked me to bug-proof it for the future.

File: src/test/java/com/google/sps/data/DatastoreAvailabilityDaoTest.java (hunk 1 above)
Comment (meagancheese) [reply src/test/java/com/google/sps/data/DatastoreAvailabilityDaoTest.java]: Done

**Review by GrantComm — APPROVED** [review]
LGTM

**Review by acmcarther — APPROVED** [review]

**Pull request merged**